### PR TITLE
Change Configuration

### PR DESCRIPTION
**Pull request by dbalencar**
Project Level Merge Request Approvals
https://github.com/xanzy/go-gitlab/issues/1340
https://docs.gitlab.com/ee/api/merge_request_approvals.html#change-configuration

### CODE REVIEW

File: merge_request_approvals.go [hunk 1]
```diff
@@ -35,31 +35,36 @@ type MergeRequestApprovalsService struct {
 //
 // GitLab API docs:
 // https://docs.gitlab.com/ee/api/merge_request_approvals.html#merge-request-level-mr-approvals
+// https://docs.gitlab.com/ee/api/merge_request_approvals.html#change-configuration
```
Comment (svanharmelen) [inline merge_request_approvals.go:38]: Please remove this line, don't think we need more then one link for one struct.

File: merge_request_approvals.go [hunk 2]
```diff
@@ -417,3 +422,28 @@ func (s *MergeRequestApprovalsService) DeleteApprovalRule(pid interface{}, merge
 
 	return s.client.Do(req, nil)
 }
+
+// ChangeProjectMergeRequestApprovalConfiguration updates the merge request approval configuration of a project.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/merge_request_approvals.html#change-configuration
+func (s *MergeRequestApprovalsService) ChangeProjectApprovalConfiguration(pid interface{}, opt *MergeRequestApprovals, options ...RequestOptionFunc) (*MergeRequestApprovals, *Response, error) {
```
Comment (svanharmelen) [inline merge_request_approvals.go:430]: This is not correct. `MergeRequestApprovals` is not an option struct so should only be used as return value. Reading the docs it seems this only allows you to change the number of approvals required and so it should probably get its own option struct.

**Review by svanharmelen — CHANGES_REQUESTED** [review]

File: merge_request_approvals.go (hunk 2 above)
Comment (dbalencar) [reply merge_request_approvals.go:430]: I agree, I can definitely make the required changes... Would you like to suggest a name for this new option struct, please?

File: merge_request_approvals.go (hunk 1 above)
Comment (dbalencar) [reply merge_request_approvals.go:38]: No problem, I'll do it when I create the new option struct.

File: merge_request_approvals.go (hunk 2 above)
Comment (svanharmelen) [reply merge_request_approvals.go:430]: Please check the other code for examples, option struct always use the name of the method with the postfix `Options`. So in this case that should be `ChangeProjectApprovalConfigurationOptions`.

**Comment by svanharmelen** [comment]
As far as I can see this is already implemented here: https://github.com/xanzy/go-gitlab/blob/master/projects.go#L1688-L1725

Is that correct or am I missing something? If not, I suggest to close this PR.

**Comment by svanharmelen** [comment]
Closing due to no response. So for now I'm going to assume this is already fixed and no longer needed. If that is incorrect, feel free to reopen this one.

**Pull request closed without merging**